<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QWidget" name="SplashScreen">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Spyder Player - SpashScreen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(216, 216, 216);</string>
  </property>
  <widget class="QLabel" name="Background_image">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>-30</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>2000</width>
     <height>2000</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/icons/icons/spider-black.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Title_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SPYDER Player</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Version_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Version 1.0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Status_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background:  rgba(255, 255, 255, 64);
	/*border-radius: 10px;*/
	color: white;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>-60</y>
     <width>531</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*background-color: qlineargradient(spread:pad, x1:0.074, y1:0.968409, x2:0.881759, y2:0.221591, stop:0 rgba(209, 30, 255, 255), stop:1 rgba(37, 189, 255, 255));*/

/*background-color: rgb(20, 5, 37);  */

background-color: qlineargradient(spread:pad, x1:0.369, y1:0.182, x2:1, y2:1, stop:0 rgba(57, 9, 126, 255), stop:1 rgba(129, 18, 129, 255));</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>Background_image</zorder>
  <zorder>Title_label</zorder>
  <zorder>Version_label</zorder>
  <zorder>Status_label</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
